blocksWorld/polygon.py

```diff
@@ -4,7 +4,8 @@
 
 import random
 
-from localTypes import *
+from blocksWorld import *
```

Comment:
I think this should be avoided. bloksWorld is the package, it should not include itself.